ucd-parse/src/arabic_shaping.rs

```diff
@@ -24,7 +24,114 @@ pub struct ArabicShaping {
     /// The "joining type" of this codepoint.
     pub joining_type: JoiningType,
     /// The "joining group" of this codepoint.
-    pub joining_group: String,
+    pub joining_group: JoiningGroup,
```

Comment:
This is a breaking change. Breaking changes should be called out in the PR description.

Could we please back this out? I'd really prefer not to release a new semver incompatible version of `ucd-parse` at this point. I could, but I would prefer such things be deliberate instead of a reactive "oh this PR came in with a breaking change so let's just put out an incompatible release." That's how churn happens. :-(

Reply:
An alternative here is to add a new field that is an enum, and leave the existing string field. Maybe `joining_group2`?

Reply:
> Breaking changes should be called out in the PR description.

Sorry I didn't think about the impact of the change when doing it so didn't think to call it out in the PR.